src/metatrain/experimental/nanopet/modules/nef.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+
+import torch
+
+
+# NEF stands for Node Edge Feature, representing the three dimensions of
+# the internal representations of a PET model. This module contains
+# functions to manipulate these representations.
+
+
+def get_nef_indices(centers, n_nodes: int, n_edges_per_node: int):
+    # computes tensors of indices useful to convert between edge
+    # and NEF layouts
```

Comment:
Could you make what these "tensors of indices" contain more explicit? i.e. for each of `edges_to_nef, nef_to_edges_neighbor, nef_mask`, add a quick explanation?